### ISSUE TITLE
Missing Test Case - 12. Integer to Roman

### ISSUE THREAD

**Issue by hellosugaree** [LeetCode-Feedback/LeetCode-Feedback#2677]
#### Category of the bug
- [ ] Question
- [x] Solution
- [ ] Language

#### Code you used for Submit/Run operation

```var map = {
  1000: 'M',
  900: 'CM',
  500: 'D',
  400: 'CD',
  100: 'C',
  90: 'XC',
  50: 'L',
  40: 'XL',
  10: 'X',
  9: 'IX',
  5: 'V',
  4: 'IV',
  1: 'I'
};

var intToRoman = function (num) {
  let roman = '';
  while (num > 0){
    let largestKey = Math.max(...Object.keys(map));
    if (num >= largestKey){
      num -= largestKey;
      roman += map[largestKey];
    } else {
      delete map[largestKey];
    }
    if (num === 0) return roman;
  }
}
```

**Comment by lc-kay** [comment]
@hellosugaree Thanks for reaching out to us. What is the content-related issue you'd like to report?

**Comment by hellosugaree** [comment]
Sorry, for some reason the info I put in the ticket didn't show up here. My original request was related to the fact that one of the test cases failed on submitting the code, yet when I run the code in my IDE or run the code using a custom testcase directly on the website it passes with the right result. I entered the specific test case info when I filled out the ticket, but clearly I didn't enter it in the right place because it never came through. At this point I don't remember what the test case was, so closing the ticket.
